### PR TITLE
Normalize differences between master and rel/beta

### PR DESCRIPTION
**Pull request by onetechnical**
## Summary

Normalizing differences between master and rel/beta branches. Some of these differences may be a result of the auto-merge.

## Test Plan

N/A

### CODE REVIEW

File: daemon/algod/api/client/restClient.go [hunk 1]
```diff
@@ -398,12 +398,6 @@ func (client RestClient) Catchup(catchpointLabel string) (response privateV2.Cat
 	return
 }
 
-// RawBlock gets the encoded, raw msgpack block for the given round
-func (client RestClient) RawBlock(round uint64) (response v1.RawBlock, err error) {
-	err = client.getRaw(&response, fmt.Sprintf("/block/%d", round), rawblockParams{1})
-	return
-}
-
```
Comment (tsachiherman) [inline daemon/algod/api/client/restClient.go:406]: would that even compile without this function?

File: ledger/ledger.go [hunk 2]
```diff
@@ -130,7 +130,6 @@ func OpenLedger(
 
 	err = l.reloadLedger()
 	if err != nil {
-		err = fmt.Errorf("OpenLedger.bqInit %v", err)
```
Comment (tsachiherman) [inline ledger/ledger.go:133]: this one is ok to remove.

File: test/scripts/e2e_go_tests.sh [hunk 3]
```diff
@@ -89,15 +89,6 @@ fi
...
 
-# ARM64 has some memory related issues with fork. Since we don't really care
-# about testing the forking capabilities, we're just run the tests one at a time.
-EXECUTE_TESTS_INDIVIDUALLY="false"
-ARCHTYPE=$("${SRCROOT}/scripts/archtype.sh")
-if [ "${ARCHTYPE}" = "arm64" ]; then
-    EXECUTE_TESTS_INDIVIDUALLY="true"
-fi
-
-
```
Comment (tsachiherman) [inline test/scripts/e2e_go_tests.sh:100]: I'm pretty sure that we want to keep this one.

File: scripts/travis/after_build.sh [hunk 4]
```diff
@@ -10,15 +10,10 @@
...
 OS=$("${SCRIPTPATH}/../ostype.sh")
-ARCH=$("${SCRIPTPATH}/../archtype.sh")
 if [[ "${OS}" == "darwin" ]]; then
     # do not run these on darwin
     exit;
 fi;
-if [[ "${ARCH}" == "arm64" ]]; then
-    # do not run these on arm64
-    exit;
-fi;
```
Comment (tsachiherman) [inline scripts/travis/after_build.sh:21]: I think that we want to keep these.

File: daemon/algod/api/client/restClient.go (hunk 1 above)
Comment (algojohnlee) [reply daemon/algod/api/client/restClient.go:406]: Yes - the function exists higher up in the file, so was probably a merge issue.

File: scripts/travis/run_tests.sh [hunk 5]
```diff
@@ -16,7 +15,7 @@ if [ "${BUILD_TYPE}" = "integration" ]; then
     export SHORTTEST 
     ./test/scripts/run_integration_tests.sh
 elif [ "${TRAVIS_EVENT_TYPE}" = "cron" ] || [[ "${TRAVIS_BRANCH}" =~ ^rel/ ]]; then
-    if [ "${OS}" != "darwin" ] && [ "${ARCH}" != "arm64" ]; then
```
Comment (tsachiherman) [inline scripts/travis/run_tests.sh:19]: I think we want to keep this file as is.

File: scripts/travis/after_build.sh (hunk 4 above)
Comment (algojohnlee) [reply scripts/travis/after_build.sh]: Reverted

File: scripts/travis/run_tests.sh (hunk 5 above)
Comment (algojohnlee) [inline scripts/travis/run_tests.sh:19]: I'm presuming we'll want to revert this one too.

File: test/scripts/e2e_go_tests.sh (hunk 3 above)
Comment (algojohnlee) [reply test/scripts/e2e_go_tests.sh]: Reverted

File: scripts/travis/after_build.sh (hunk 4 above)
Comment (onetechnical) [reply scripts/travis/after_build.sh:21]: Reverted again. After discussion with Tsachi, we think matching master makes sense.

File: scripts/travis/run_tests.sh (hunk 5 above)
Comment (onetechnical) [reply scripts/travis/run_tests.sh:19]: Leaving as is, but we may  want to fix in master.

File: test/scripts/e2e_go_tests.sh (hunk 3 above)
Comment (onetechnical) [reply test/scripts/e2e_go_tests.sh:100]: And unreverted.

**Pull request merged**